### ISSUE TITLE
Guys, I created a online document site to reduce the time for reviewing information.

### ISSUE THREAD

**Issue by Abbotton** [amzn/selling-partner-api-models#87]
As the title suggests, this site can save a lot of time, you can use this site for free, or you can clone the repository and build your own.

[View Demo](https://spapi.mijing.io)
[Source code](https://github.com/mibrsoft/amazon-spapi-document)

**Comment by svickers** [comment]
A+! How does the documentation only exist in swagger form?  Very hard to get started currently or even get a sense of what endpoints are available.

**Comment by ianrussel** [comment]
This is actually nice. As it fetch he amazon repo, this will guarantee up-to date document

**Comment by mzeyen** [comment]
Thats a great idea. 👍

**Comment by Abbotton** [comment]
> A+! How does the documentation only exist in swagger form? Very hard to get started currently or even get a sense of what endpoints are available.

Yes, if you have docked MWS before, it should be easy to get started when using SPAPI, otherwise, it is really difficult to get started.

**Comment by Abbotton** [comment]
> This is actually nice. As it fetch the amazon repo, this will guarantee up-to date document

Yes, manually maintaining such an online document project is very hard. Thank you for your affirmation.

**Comment by Abbotton** [comment]
> Thats a great idea. 👍

Thank you, I hope that the online document can give you substantive help.

**Comment by jimmy-ross-xapix** [comment]
really useful, thanks!
